configs/components/openssl.rb

```diff
@@ -1,7 +1,28 @@
 component "openssl" do |pkg, settings, platform|
   pkg.version "1.0.2e"
-  pkg.md5sum "5262bfa25b60ed9de9f28d5d52d77fc5"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/openssl-#{pkg.get_version}.tar.gz"
+  if platform.is_windows?
```

Comment:
This is still really weird to me. I wish I knew why it changed. Did it start happening with 1.0.2e?